src/core/qgsvectorlayer.h

```diff
@@ -504,6 +504,25 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
 
     };
 
+    /**
+     * Context for deleting features
+     * \since QGIS 3.14
+     */
+    struct  DeleteContext
```

Comment:
```suggestion
    struct CORE_EXPORT DeleteContext
```

Reply:
@jef-n compensation for the blood, sweat and tears, nightly builds instead of a "fix windows build" commit tomorrow.